Documentation/network/kubernetes/ciliumendpointslice_beta.rst

```diff
@@ -1,117 +0,0 @@
-CiliumEndpointSlice (beta)
-***************************
-
-.. include:: ../../beta.rst
-
-This document describes CiliumEndpointSlice (CES), which enables batching of
-CiliumEndpoint (CEP) objects in the cluster to achieve better scalability.
-The concept of CiliumEndpointSlice is described in:
-
-.. toctree::
-   :maxdepth: 1
-   :glob:
-
-   ciliumendpointslice
-
-When enabled, Cilium operator watches CEP objects and group/batch slim versions
-of them into CES objects. Cilium agent watches CES objects to learn about
-remote endpoints in this mode. API-server stress due to remote endpoint info
-propagation should be reduced in this case, allowing for better scalability,
-at the cost of potentially longer delay before identity of new endpoint is
-recognized throughout the cluster.
-
-.. note::
-
-   CiliumEndpointSlice is a concept that is specific to Cilium and is not
-   related to `Kubernetes' EndpointSlice`_. Although the names are similar, and
-   even though the concept of slices in each feature brings similar
-   improvements for scalability, they address different problems.
-
-   Kubernetes' Endpoints and EndpointSlices allow Cilium to make load-balancing
-   decisions for a particular Service object; Kubernetes' EndpointSlices offer
-   a scalable way to track Service back-ends within a cluster.
-
-   By contrast, CiliumEndpoints and CiliumEndpointSlices are used to make
-   network routing and policy decisions. So CiliumEndpointSlices focus on
-   tracking Pods, batching CEPs to reduce the number of updates to propagate
-   through the API-server on large clusters.
-
-   Enabling one does not affect the other.
-
-.. _Kubernetes' EndpointSlice: https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/
-
-Deploy Cilium with CES
-=======================
-The CES feature relies on use of CEP. This feature is disabled by default
-in current beta status. You can enable the feature by setting the
-``enableCiliumEndpointSlice`` value to ``true``.
-
-.. parsed-literal::
-
-   helm install cilium |CHART_RELEASE| \\
-     --set enableCiliumEndpointSlice=true
-
-Verify that Cilium agent and operator pods are running.
-
-.. code-block:: shell-session
-
-   $ kubectl get pods -n kube-system -l k8s-app=cilium
-   NAMESPACE     NAME           READY   STATUS    RESTARTS   AGE
-   kube-system   cilium-7tnkq   1/1     Running   0          1d
-
-   $ kubectl get pods -n kube-system -l io.cilium/app=operator
-   NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
-   kube-system   cilium-operator-855979b54b-w8rsm   1/1     Running   0          1d
-
-Validate that the CiliumEndpointSlice CRD has been registered.
-
-.. code-block:: shell-session
-
-   $ kubectl get crds --all-namespaces
-   NAME                                         CREATED AT
-   [...]
-   ciliumendpointslices.cilium.io               2021-11-05T05:41:28Z
-
-Known Issues and Workarounds
-============================
-
-Potential Upgrade Impact
```

Comment:
I understand the contents from this subsection have been folded into the “migration procedure”, right?

Reply:
Yes!